library/network-tools/curl/http3/8.11/Dockerfile

```
@@ -0,0 +1,17 @@
+FROM alpine/curl-http3:8.11.0 AS curl-http3
```

```suggestion
FROM alpine/curl-http3:8.11.0 AS build
```

Use `build` everywhere.